src/common/AudioStreamBuilder.cpp

```diff
@@ -26,6 +26,19 @@
 
 namespace oboe {
 
+/**
+ * The following default values are used when oboe does not have any better way of determining the optimal values
+ * for an audio stream. The most common scenarios for this are:
+ *
+ * - Creating a stream on API < 26 (OpenSLES) but has not supplied the optimal sample
```

Comment:
The word "has" sounds strange. What has?

Reply:
Text updated.